content/guides.md

```diff
@@ -24,7 +24,7 @@ To see ports, keyboard layouts, function keys, and product quickstart guides, fi
 
 | Laptops | Desktops | Accessories |
 |:-------:|:--------:|:-----------:|
-| Adder WS [(addw1)](https://tech-docs.system76.com/models/addw1/README.html) | Meerkat [(meer5)](https://tech-docs.system76.com/models/meer5/README.html) |
+| Adder WS [(addw1)](https://tech-docs.system76.com/models/addw1/README.html) | Meerkat [(meer5)](https://tech-docs.system76.com/models/meer5/README.html) | Launch [(launch_1)](https://tech-docs.system76.com/models/launch_1/repairs.html) |
```

Comment:
The rest of the models link to `README.html`, not the repairs page. Do you want to link to https://tech-docs.system76.com/models/launch_1/README.html instead?

Reply:
Yes. Lol.